sanic_openapi/oas3/types.py

```diff
@@ -185,7 +184,7 @@ class Array(Schema):
     uniqueItems: False
 
     def __init__(self, items: Schema, **kwargs):
```

Comment:
typing for 'items' can be set to 'Any' or removed now, nice changes :)

Reply:
You're right @artcg , done!